pkg/crc/machine/start.go

```diff
@@ -369,8 +369,10 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 	}
 
 	// Trigger disk resize, this will be a no-op if no disk size change is needed
-	if err := growRootFileSystem(sshRunner); err != nil {
-		return nil, errors.Wrap(err, "Error updating filesystem size")
+	if !vm.bundle.IsMicroshift() {
+		if err := growRootFileSystem(sshRunner); err != nil {
+			return nil, errors.Wrap(err, "Error updating filesystem size")
+		}
```

Comment:
Hopefully this is only a temporary limitation ?

Reply:
@cfergeau yes, it should be temporary we need to find a way to extend the lvm with resized pv. I was able to do something following but that need `growpart` package to be installed and we might need a different helper for microshift preset for handing the resize.

```
   16  growpart /dev/vda 3
   17  lsblk 
   18  df -h
   19  pvs
   20  pvresize /dev/vda3
   21  pvs
   22  lvextend -r -l +100%FREE /dev/mapper/rhel-root 
```

Reply:
Maybe add a `// growRootFileSystem does not support LVM which is used by microshift images`